[2022-07-29 11:42:16,504] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-07-29 11:42:16,524] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator.get_name scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-07-29 11:42:16,524] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:42:16,524] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 11:42:16,525] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 11:42:16,547] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_name> on 2022-01-04 00:00:00+00:00
[2022-07-29 11:42:16,553] {standard_task_runner.py:52} INFO - Started process 6273 to run task
[2022-07-29 11:42:16,560] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator', 'get_name', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '675', '--raw', '--subdir', 'DAGS_FOLDER/CreateDagWithPythonOperator.py', '--cfg-path', '/tmp/tmpcd6vm1zi', '--error-file', '/tmp/tmpiugaw0nk']
[2022-07-29 11:42:16,561] {standard_task_runner.py:80} INFO - Job 675: Subtask get_name
[2022-07-29 11:42:16,645] {task_command.py:369} INFO - Running <TaskInstance: dag_with_python_operator.get_name scheduled__2022-01-04T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 11:42:16,780] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=dag_with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-01-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-04T00:00:00+00:00
[2022-07-29 11:42:16,835] {python.py:173} INFO - Done. Returned value was: None
[2022-07-29 11:42:16,891] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator, task_id=get_name, execution_date=20220104T000000, start_date=20220729T114216, end_date=20220729T114216
[2022-07-29 11:42:16,940] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 11:42:16,991] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
